### PR TITLE
Allow for reporting of "map_files" paths from normalization APIs

### PR DESCRIPTION
**Pull request by danielocfb**

### CODE REVIEW

File: benches/normalize.rs [hunk 1]
```diff
@@ -19,13 +20,21 @@ where
...
+    let opts = NormalizeOpts {
+        sorted_addrs: true,
+        ..Default::default()
+    };
 
     let () = b.iter(|| {
         let normalized = normalizer
-            .normalize_user_addrs_sorted(black_box(0.into()), black_box(addrs.as_slice()))
+            .normalize_user_addrs_opts(
+                black_box(0.into()),
```
Comment (anakryiko) [inline benches/normalize.rs:34]: curious, why using black_box() here?

**Review by anakryiko — APPROVED** [review]
LGTM.

File: benches/normalize.rs (hunk 1 above)
Comment (d-e-s-o) [reply benches/normalize.rs:34]: We don't want the optimizer to inline everything and potentially specialize the code for the arguments in question -- which are available at compile/link time -- as that could perturb the reported numbers in a positive way not necessarily representative of real world usage (where data may only be known at runtime).

**Comment by d-e-s-o** [comment]
Thanks for the review! I think we are still undecided whether we actually need this option. So I'll go ahead and land the first commit only for now. May also differentiate between `blaze_normalize_user_addrs` and `blaze_normalize_user_addrs_opts` in the C API.

**Comment by d-e-s-o** [comment]
> I think we are still undecided whether we actually need the `map_files` option. So I'll go ahead and land the first commit only for now.

https://github.com/libbpf/blazesym/pull/704

**Comment by d-e-s-o** [comment]
I think there is a need for this option, because it allows for access of deleted files while the process is still running, but making that behavior the default is probably not the right approach. So let's go ahead with the merge.

**Pull request merged**